paying_for_college/tests/test_scripts.py

```diff
@@ -35,15 +35,20 @@ class PurgeTests(django.test.TestCase):
     fixtures = ['test_fixture.json', 'test_program.json']
 
     def test_purges(self):
+        self.assertFalse(Program.objects.count() == 0)
```

Comment:
The Django documentation suggests using [`exists`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#exists) instead when checking for the existence of queries, so you could do something like:

```python
self.assertFalse(Notification.objects.exists())
```